src/adbg/dumper/objs/printpe.d

```diff
@@ -787,4 +876,11 @@ L_DISASM:
 	}
 
 	return EXIT_SUCCESS;
-}
\ No newline at end of file
+}
+
+private void print_guid(GUID guid) {
```

Comment:
I think it would be much more worthwhile of adding a `utils.uid` module that deals with both GUIDs and UUIDs. A `uid_str(ref UID uid, char[UID_STR_LENGTH] buf, int target)` function should be able to to format the string buffer and the `uid_swap(ref UID uid)` function should swap the endianness if the system endianness does not match the GUID (being LSB)/UUID (being MSB).